app/models/organization.rb

```diff
@@ -81,6 +82,19 @@ def bottom(limit = 5)
   has_many :users, dependent: :destroy
   has_many :requests, dependent: :destroy
   has_many :audits, dependent: :destroy
+  before_update :update_partner_sections, if: :partner_form_fields_changed?
```

Comment:
```suggestion
  after_update :update_partner_sections, if: :partner_form_fields_changed?
```

Reply:
This was actually intentionally left as before_update -- I don't think we can use the rails `_changed?` method on an after_update because it has been changed in the DB at that point.